src/confdb/confdb.c

```diff
@@ -872,7 +872,7 @@ static int confdb_get_domain_internal(struct confdb_ctx *cdb,
     struct sss_domain_info *domain;
     struct ldb_result *res;
     TALLOC_CTX *tmp_ctx;
-    const char *tmp;
+    const char *tmp, *tmp2;
```

Comment:
Maybe you can use a bit more verbose name e.g. tmp_pam_target?

Reply:
Done.